DataManagementSystem/DB/FileCatalogComponents/test/Test_DataManagement_FileCatalogComponents.py

```diff
@@ -0,0 +1,101 @@
+""" Test class for FileCatalogComponents
```

Comment:
Dont' you want to remove that ? It seems useless no ? I thought it was just to try to make codecov happy.

Reply:
It is not super useful but at least it covers few lines of code, and clearly checks it there are no circular imports. So, I would leave it.